app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -97,6 +98,8 @@ fun ImportFilesScreen(
 
     LaunchedEffect(Unit) { importFilesViewModel.initTransferOptionsValues() }
 
+    BackHandler { closeActivity() } // Closing the activity like other screens will prompt the user to be sure he wants to leave
```

Comment:
I was a bit confuse with this sentence at first

Maybe we could rewrite this along the line of "We need to overwrite this to avoid prompting the user to be sure he wants to leave" ?